packages/closer/locales/base-pt.json

```
@@ -1575,5 +1588,107 @@
+  "volunteers_recommended_stay": "A estadia recomendada é de 1 mês - e idealmente chegando no início do mês para que possamos ter uma boa dinâmica de grupo.",
+  "projects_builders_residency_open_call": "Chamada Aberta para Residência de Construtores",
+  "projects_starting_to_build": "Estamos a começar a construir! A TDF garantiu financiamento para iniciar grandes renovações. Em janeiro de 2025, a primeira fase de alojamento começará com um empreiteiro profissional e, paralelamente, estaremos a auto-construir vários projetos para apoiar a abertura da TDF V2.",
+  "projects_requirements_label": "Requisitos:",
+  "projects_requirements_value": "6 horas por dia, mínimo de 1 mês ✅. Alojamento e alimentação gratuitos (tendas glamping e dormitórios disponíveis, ou traga a sua carrinha)",
+  "projects_token_rewards_label": "Recompensas em tokens por projetos concluídos!💰",
+  "projects_token_rewards_value": "Ofereceremos tokens aos construtores que levarem projetos do início ao fim. As recompensas serão negociadas com a equipa.",
+  "projects_community_culture_label": "Cultura comunitária:",
+  "projects_community_culture_value": "Viva em co-living com outros construtores e a equipa TDF. Teremos atividades comunitárias regulares, saunas, jantares experienciais, jams de música, práticas de embodiment, yoga, massagem e muito mais 🥙💃🏽🔥🎶🎭",
+  "projects_build_projects_title": "Projetos de Construção 🛠🏡🛕",
+  "projects_skill_practical_build": "Competências práticas de construção",
+  "projects_skill_electrical": "Eletricidade",
```

Biome (1.9.4)

[error] The key volunteers_recommended_stay was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)